tools/xchain-suite/src/components/trade/AssetBalanceCard.tsx

```
@@ -14,7 +14,7 @@ const ASSET_TO_CHAIN: Record<string, string> = {
   BTC: 'BTC', ETH: 'ETH', SOL: 'SOL', AVAX: 'AVAX', DOGE: 'DOGE',
   LTC: 'LTC', BCH: 'BCH', XRP: 'XRP', ADA: 'ADA', BNB: 'BSC',
   ARB: 'ARB', SUI: 'SUI', GAIA: 'GAIA', THOR: 'THOR', MAYA: 'MAYA',
-  KUJI: 'KUJI', DASH: 'DASH', ZEC: 'ZEC', XRD: 'XRD', XMR: 'XMR',
+  DASH: 'DASH', ZEC: 'ZEC', XRD: 'XRD', XMR: 'XMR', NEAR: 'NEAR',
```

ESLint

[error] Replace `·ZEC:·'ZEC',·XRD:·'XRD',·XMR:·'XMR',` with `⏎··ZEC:·'ZEC',⏎··XRD:·'XRD',⏎··XMR:·'XMR',⏎·`

(prettier/prettier)